schedule/qam/12-SP2/qam-minimal.yaml

```diff
@@ -7,6 +7,7 @@ schedule:
 - {{disk_activation}}
 - installation/scc_registration
 - installation/addon_products_sle
+- {{system_role}}
```

Comment:
FYI, we have figured out that curly braces are special symbols in yaml, so it's recommended to wrap it in quotes.

Reply:
lame ... well I guess I will do it in separate PR for all QAM yamls then